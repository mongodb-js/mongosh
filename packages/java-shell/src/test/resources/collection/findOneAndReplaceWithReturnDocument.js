// before
db.coll.remove({});
db.coll.insertOne({a: 1});
// command
db.coll.findOneAndReplace({a: 1}, {a: 2}, {returnNewDocument: true});
// command
db.coll.find();
// clear
db.coll.drop();
