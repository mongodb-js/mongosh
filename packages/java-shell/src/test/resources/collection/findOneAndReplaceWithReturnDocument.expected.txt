{ "_id": <ObjectID>, "a": 2 }
[ { "_id": <ObjectID>, "a": 2 } ]